maps_dashboards/server/saved_objects/map_saved_object.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const mapSavedObjectsType: SavedObjectsType = {
+  name: 'map',
+  hidden: false,
+  namespaceType: 'agnostic',
+  management: {
+    defaultSearchField: 'title',
+    importableAndExportable: true,
+    getTitle(obj) {
+      return obj.attributes.title;
+    },
+    getInAppUrl(obj) {
+      return {
+        path: `/app/maps-dashboards#/${encodeURIComponent(obj.id)}`,
+        uiCapabilitiesPath: 'map.show',
+      };
+    },
+    getEditUrl(obj) {
+      return `/management/opensearch-dashboards/objects/map/${encodeURIComponent(obj.id)}`;
+    },
+  },
+  mappings: {
+    properties: {
```

Comment:
2 other properties you might want to consider
1. version: useful to teack the version differences in your saved object mapping
2. kibanaSavedObjectMeta: To reference other saved objects

e.g. https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_builder/server/saved_objects/vis_builder_app.ts#L50-L56

Reply:
Thanks the adding, added.